### PR TITLE
my first commit

### PR DESCRIPTION
**Pull request by tdhankhar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "twitch",
   "version": "1.0.0",
-  "description": "Streaming Platform",
+  "description": "Best streaming platform in the world",
```
Comment (HardeepSaggu) [inline package.json:4]: Let's make P capital for platform

Comment (tdhankhar) [reply package.json:4]: Will take care of it

**Review by HardeepSaggu — COMMENTED** [review]
LGTM

**Review by HardeepSaggu — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "twitch",
   "version": "1.0.0",
-  "description": "Streaming Platform",
```
Comment (Achintnarang3) [inline package.json:4]: Change

**Pull request closed without merging**